addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -150,7 +150,7 @@ function factory(dependencies) {
          * is not yet initialized.
          */
         openInitThread() {
-            const [model, id] = this.initActiveId.split('_');
+            const [model, id] = typeof this.initActiveId === 'number' ? ['mail.channel', this.initActiveId] : this.initActiveId.split('_');
```

Comment:
Also add a comment on why/which case the "number" value is given.

Reply:
It should be explained as docstring of `activeId` fields.